packages/orchestration/src/exos/orchestrator.js

```diff
@@ -72,15 +74,15 @@ export const prepareOrchestratorKit = (
     {
       orchestrator: OrchestratorI,
       makeLocalChainFacadeWatcher: M.interface('makeLocalChainFacadeWatcher', {
-        onFulfilled: M.call(M.record())
-          .optional(M.arrayOf(M.undefined()))
+        onFulfilled: M.call(M.record(), M.string())
+          .optional(M.arrayOf(M.undefined())) // XXX needed?
```

Comment:
I don't know but please address before merge

Reply:
seems to be not needed